backend/authentication/password_validators.py

```diff
@@ -7,6 +7,7 @@
 
 has_alphabetic = re.compile(r"[a-zA-Z]")
 has_numeric = re.compile(r"[\d]")
+min_score = 3
```

Comment:
so this is on me, it should be `MIN_SCORE` (all caps) since it's a global constant. The `has_alphabetic` and `has_numeric` could/arguably should but don't bother changing it. yours should be all caps though.

pep8 defines the style we keep to, here's what it says about constants like this: https://www.python.org/dev/peps/pep-0008/#constants